### PR TITLE
Make sidesheets sticky

### PR DESCRIPTION
**Pull request by nicholasdalhaug**

### CODE REVIEW

File: frontend/src/views/Evaluation/EvaluationView.tsx [hunk 1]
```diff
@@ -64,22 +64,26 @@ const EvaluationView = ({evaluation, onProgressEvaluationClick, onProgressPartic
                 description={calcProgressionStatus(evaluation.progression, Progression.Workshop)}
                 stepKey={Progression.Workshop}
             >
-                <WorkshopView
-                    evaluation={evaluation}
-                    onNextStepClick={() => onProgressEvaluationClick()}
-                    onProgressParticipant={onProgressParticipant}
-                />
+                <div>
```
Comment (nicholasdalhaug) [inline frontend/src/views/Evaluation/EvaluationView.tsx:67]: I have placed a div here and below to make the `position: sticky` work. Not sure exactly why that fixed it 😬

Comment (Christdej) [reply frontend/src/views/Evaluation/EvaluationView.tsx:67]: Is it worth adding a comment about this?

Comment (nicholasdalhaug) [reply frontend/src/views/Evaluation/EvaluationView.tsx:67]: Done

**Review by Christdej — APPROVED** [review]

File: frontend/src/views/Evaluation/EvaluationView.tsx (hunk 1 above)
Comment (denektenina) [reply frontend/src/views/Evaluation/EvaluationView.tsx:67]: I like seeing some comments if something isn't obvious :)

**Review by denektenina — APPROVED** [review]

**Pull request merged**